CLI/turBOT.py

```diff
@@ -111,38 +137,51 @@ def parse_filename(file_path):
     return prefix
 
 def main():
-    parser = argparse.ArgumentParser(description='Binary Obfuscation Tool (BOT), default flags -DRO')
+    parser = argparse.ArgumentParser(description='Binary Obfuscation Tool (BOT). The flags D, R and O are enabled by default')
     parser.add_argument('-D', action='store_const', const=True, default=False, help='Disassembles a specified binary')
```

Comment:
I don't know if it's just a me thing but do we want these flags to be lowercase?

Reply:
Honestly, I agree but I was wary about changing anything too much that might of come up in the usability testing because I didn't want to step on anybody's toes

Reply:
Ah good point then feel free to ignore this